Documentation/how-to-review-a-pr.md

```diff
@@ -0,0 +1,70 @@
+# PR Review Policy
+
+To ensure code quality, maintainability, and stability, all Pull Requests (PRs) follow a three-step review process. This process includes automated checks followed by two distinct types of human review.
+
+## Automated: 1st Step
+Before any human review, all PRs must successfully pass a series of automated checks. These tools provide the first line of defense against common issues.
+
+### Linter & Formatter
+Automatically fixes all style and formatting issues.
+
+We enforce a shared style guide (for example, Mantra will reuse the style guide from nt-gui).
+
+### Test Coverage
+Ensures that tests are present to prove the code works as intended.
+
+
+### (future) Add Static Analysis
+This is a more long-term goal for additional risk reduction.
+
+A tool like SonarQube or Gitlab SATS will be used to scan the PR for code smells, complexity issues, and potential bugs. Some static code analysis can also be done within the Linter & Formatter step.
+
+### Type Script (Frontend Specific)
```

Comment:
What about mentioning other cases like mypy for Python? Or golangci-lint for go?